La'Tonia/coin_toss.py

```diff
@@ -0,0 +1,48 @@
+# Yusuf provided this code ... I did not code this code ... fun to run
+import random
+from tqdm import trange, tqdm
```

Comment:
This is a module I never installed or worked with, but I regret it. Something very useful!